app/assets/javascripts/discourse/models/composer.js.es6

```diff
@@ -709,6 +709,11 @@ const Composer = RestModel.extend({
       const topicProps = this.getProperties(
         Object.keys(_edit_topic_serializer)
       );
+      // frontend should have featuredLink but backend needs featured_link
+      if (typeof topicProps.featuredLink !== "undefined") {
```

Comment:
Any reason why you didn't do `if (topicProps.featuredLink)` here?

Reply:
No particular reason, I have changed it to `if (topicProps.featuredLink)` :bowing_man: